treeherder/etl/perf_data_adapters.py

```diff
@@ -235,6 +234,48 @@ def adapt_and_load(self, reference_data, job_data, datum):
             result_set_id = job_data[_job_guid]['result_set_id']
             push_timestamp = job_data[_job_guid]['push_timestamp']
 
+            # counters will not be part of the summary series
+            # counters have a json obj {'stat': val} instead of [val1, val2, ...]
+            for _test in talos_datum["talos_counters"].keys():
+                signature_properties = {
+                    'suite': _suite,
+                    'test': _test
+                }
+                signature_properties.update(reference_data)
+
+                series_signature = self._get_series_signature(
+                    signature_properties)
+
+                series_data = {
+                    "job_id": job_id,
+                    "result_set_id": result_set_id,
+                    "push_timestamp": push_timestamp,
+                    "total_replicates": 1,
+                    "min": 0,
+                    "max": 0,
+                    "mean": 0,
+                    "std": 0,
+                    "median": 0
+                }
+                for stat in ['min', 'max', 'mean', 'median', 'std', 'total_replicates']:
+                    if stat in talos_datum["talos_counters"][_test]:
+                        # in case we have non int/float data, lets ignore it
```

Comment:
It's pretty easy, we don't do it in this file, but we do elsewhere. Just import logging, get a logger object, and then use it as needed. See e.g.:

https://github.com/mozilla/treeherder/blob/master/treeherder/etl/oauth_utils.py#L11
https://github.com/mozilla/treeherder/blob/master/treeherder/etl/oauth_utils.py#L61

The warning would appear in the treeherder log (/var/log/treeherder/treeherder.log). Or once we switch to heroku, it'll be in papertrail (https://papertrailapp.com/). Useful for troubleshooting!